co-authors-plus.php

```diff
@@ -380,14 +383,15 @@ public function coauthors_meta_box( $post ) {
 				<?php
 				foreach ( $coauthors as $coauthor ) :
 					$count++;
+					$avatar_url = get_avatar_url( $coauthor->ID, array( 'default' => 'gravatar_default' ) );
 					?>
 					<li>
-						<?php echo get_avatar( $coauthor->user_email, $this->gravatar_size ); ?>
 						<span id="<?php echo esc_attr( 'coauthor-readonly-' . $count ); ?>" class="coauthor-tag">
 							<input type="text" name="coauthorsinput[]" readonly="readonly" value="<?php echo esc_attr( $coauthor->display_name ); ?>" />
 							<input type="text" name="coauthors[]" value="<?php echo esc_attr( $coauthor->user_login ); ?>" />
 							<input type="text" name="coauthorsemails[]" value="<?php echo esc_attr( $coauthor->user_email ); ?>" />
 							<input type="text" name="coauthorsnicenames[]" value="<?php echo esc_attr( $coauthor->user_nicename ); ?>" />
+							<input type="text" name="coauthorsavaters[]" value="<?php echo esc_attr( $avatar_url ); ?>" />
```

Comment:
If we keep the `get_avatar()` call above, we can change the input type to `hidden` instead of `text` since it may not be necessary need to display the URL to the `avatar_url` in a text field.

Also, not a big deal, but we should fix the typo here of `avatar` so that the `name` attribute should be `coauthorsavatars`.

Reply:
I do all changes, and there is no option to hide `avatar_url ` in a text field because this plugin use [ Jquery suggest](https://web.archive.org/web/20111017233444/http://plugins.jquery.com/project/suggest) that not support to render output like jQuery Autocomplete.